instrumentation/aws-sdk/aws-sdk-2.2/agent/agent.gradle

```diff
@@ -0,0 +1,36 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+}
+
```

Comment:
Do we need extra folder layer? `instrumentation/aws-sdk/aws-sdk-2.2` vs `instrumentation/aws-sdk-2.2`

Reply:
The extra folder layer helps keep the `instrumentation` directory size down (e.g. in Intellij) which I think makes navigation easier